sample_pages/esp/eids-esp.1.0.js

```diff
@@ -0,0 +1,26 @@
+function fetchAsyncSignals(mode) {
+        switch (mode) {
+        case 1:
+          var eids=""
+          if(pbjs && pbjs.getUserIdsAsEids && typeof pbjs.getUserIdsAsEids === "function"){
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 2 tabs but found 10 spaces.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15858)